src/background_service.js

```diff
@@ -0,0 +1,26 @@
+import Constants from "./constants.js";
+import Functions from "./functions.js";
+
+const functions = new Functions();
+functions.checkUpdateUrls();
+setAlarm();
+
+chrome.tabs.onUpdated.addListener(functions.onTabUpdated.bind(functions));
+
+async function setAlarm() {
+  const alarm = await chrome.alarms.get(Constants.UPDATE_CHECK_ALARM_NAME);
+
+  if (!alarm) {
+    // Register for periodic endpoint updates
+    await chrome.alarms.create("update-sponsor-links-alarm", {
+      delayInMinutes: Constants.UPDATE_CHECK_INTERVAL,
+      periodInMinutes: Constants.UPDATE_CHECK_INTERVAL
+    });
+
+    chrome.alarms.onAlarm.addListener(function (alarm) {
+      if (alarm.name === Constants.UPDATE_CHECK_ALARM_NAME) {
+        functions.updateUrls();
```

Comment:
Dit zou bij nader inzien `checkUpdateUrls` moeten zijn. Maar dat is niet wat er op dit moment live staat. ik maak er wel een bug report van, dan kan iemand dat later een keer oppakken en dan opnieuw live zetten.

Reply:
@DrumsnChocolate Kan je issues tab aanzetten voor deze repo? ik kan op dit moment geen issues aanmaken

Reply:
Aangezet